processor/src/test/java/org/mapstruct/ap/test/nestedproperties/simple/SimpleMapper.java

```diff
@@ -31,6 +31,7 @@
     SimpleMapper MAPPER = Mappers.getMapper( SimpleMapper.class );
 
     @Mappings( { @Mapping( target = "longValue", source = "props.longValue" ),
+        @Mapping( target = "publicLongValue", source = "props.publicLongValue" ),
```

Comment:
keep it simple.. Just reverse (`@InheritInverseConfiguration`) and make sure there are no sneaky issues :).